docs/HailExpressionLanguage.md

```diff
@@ -56,6 +56,8 @@ Several Hail commands provide the ability to perform a broad array of computatio
      - max: `arr.max` -- valid only for numeric arrays, returns the minimum value
      - find: `arr.find(v => expr)` -- Returns the first non-missing element of `arr` for which `expr` is true.  If no element satisfies the predicate, `find` returns NA.
      - map: `arr.map(v => expr)` -- Returns a new array produced by applying `expr` to each element
+     - flatMap: `arr.flatMap(v => expr)` -- valid only for `expr` of type Array. Returns a new array by mapping each element of `arr` and concatenating the resulting arrays
```

Comment:
No, I was purposing it everywhere.  Is `set.toArray` well-defined?  Because we already have that.  I'm proposing the Scala model.

```
scala> Set(Array(1, 2), Array(3, 4)).flatten
res4: scala.collection.immutable.Set[Int] = Set(1, 2, 3, 4)

scala> Array(Set(1, 2), Set(3, 4)).flatten
res5: Array[Int] = Array(1, 2, 3, 4)
```

Monad consistency is also reasonable.  Question is, what is more reasonable from the user perspective?